core/libp2p/peer/address_repository.cpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "libp2p/peer/address_repository.hpp"
+
+namespace libp2p::peer {
+
+  boost::signals2::connection AddressRepository::onAddressAdded(
+      const std::function<AddressCallback> &cb) {
```

Comment:
not sure, maybe better receive by value

Reply:
![image](https://user-images.githubusercontent.com/1867551/56849775-a96eb780-6901-11e9-84c5-cfee2a3dd7ec.png)

Reply:
![image](https://user-images.githubusercontent.com/1867551/56849780-c4412c00-6901-11e9-86d0-4d8d1747ce55.png)

Reply:
Why not make it non-const?

Reply:
connect accepts argument by const ref

Reply:
Ok